### ISSUE TITLE
Meteor npm use of ftpimp

### ISSUE THREAD

**Issue by jmblanc** [sparkida/ftpimp#28]
Hi, 

i have been tested your library with meteor (meteor npm install ftpimp --save). And after I add  : var FTP = require('ftpimp'), 
I have obtained this error :  SyntaxError: Unexpected token =
W20170406-15:28:16.800(2)? (STDERR)     at Object.exports.runInThisContext (vm.js:53:16)
W20170406-15:28:16.802(2)? (STDERR)     at ...\site\.meteor\local\build\programs\server\boot.js:289:30
etc ...

this errors arrive both on windows and linux. If you can have an idea of how this arrives ?

Thank you

**Comment by sparkida** [comment]
I'll take a look, just curious, what version of node are you running with?

**Comment by jmblanc** [comment]
Thank you,
 I am using meteor 1.4.2.7, that includes a version 4.7.3 of node.

**Comment by sparkida** [comment]
@jmblanc I was able to quickly locate and fix the issue, a particular es6 feature being the culprit. Thanks again for reporting the issue and providing the context to replicate it.

***TLDR;*** `npm i ftpimp` to get the latest fix `@3.0.3`
